subprojects/gradle/build-environment/src/main/kotlin/com/avito/utils/gradle/EnvironmentInfo.kt

```diff
@@ -0,0 +1,84 @@
+package com.avito.android.sentry
+
+import com.avito.git.Git
+import com.avito.kotlin.dsl.lazyProperty
+import com.avito.logger.DefaultLogger
+import com.avito.logger.LogLevel
+import com.avito.logger.Logger
+import com.avito.logger.LoggerFactory
+import com.avito.logger.LoggingDestination
+import com.avito.logger.handler.DefaultLoggingHandler
+import com.avito.utils.gradle.Environment
+import com.avito.utils.gradle.internal.EnvironmentInfoImpl
+import org.gradle.api.Project
+import org.gradle.api.provider.Provider
+
+/**
+ * Use [Project.environmentInfo] to gain instance
+ */
+interface EnvironmentInfo { // TODO: merge with BuildEnvironment and EnvArgs
```

Comment:
Moved temporarily this class from `sentry-config` module.
It is used mostly for metrics.
I want to split it and merge if possible with other similar abstractions: BuildEnvironment and EnvArgs